kateryna/daemon/log_reader_daemon.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/sh
+SCRIPT=./parsing_logs.sh
+RUNAS=root
+PIDFILE=./my_command.pid
+LOGFILE=./my_command.log
+
+start() {
+   if [ -f "$PIDFILE" ] && kill -0 $(cat "$PIDFILE"); then
+   echo 'Service already running' >&2
+    return 1
+  fi
+  echo 'Starting service…' >&2
+  echo $$  > "$PIDFILE"
+  local CMD="$SCRIPT &> \"$LOGFILE\" & echo \$!"
+  su -c "$CMD" $RUNAS > "$PIDFILE"
```

Comment:
I found out that mistake happen when pid file is empty but i'm unaware why it may be empty, 
in my ubuntu it works fine